src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ElementType } from 'react'
+import DropdownRB from 'react-bootstrap/Dropdown'
+import DropdownButton from 'react-bootstrap/DropdownButton'
+import { Item } from './interfaces'
+import { ColorVariants } from '../../helpers/ColorVariants'
+
+interface Props extends React.Props<any> {
+  /** Determines the dropdown toggle text */
+  text: string
+  /* Determines the dropdown's items */
+  items: Item[]
+  /* Determines the dropdown toggle button id */
+  id: string
+  /* Determines the dropdown toggle variant color */
```

Comment:
I think, after looking at this a second time, id should be optional here. This would be consistent with the rest of the components.

Reply:
I had forgotten I had put them as necessary to match the React Bootstrap Dropdown.Button Dropdown.Toggle etc which ids are required. 
Shall I put it as required or is a work around necessary?